pyanaconda/ui/gui/spokes/custom_storage.py

```diff
@@ -1509,7 +1508,10 @@ def on_fs_type_changed(self, combo):
     def on_encrypt_toggled(self, widget):
         self._request.device_encrypted = self._encryptCheckbox.get_active()
         self.on_luks_version_changed(self._luksCombo)
+        self._update_luks_combo()
+        self.on_value_changed()
```

Comment:
Is it really required to have here `on_value_changed()` when this is already called by `on_luks_version_changed()`?

Reply:
I don't really want to rely on that. Yes, this call is redundant at this moment, but it can change in the future.